contracts/seed/Seed.sol

```diff
@@ -279,13 +296,22 @@ contract Seed {
     */
     function close() public onlyAdmin isActive {
         // transfer seed tokens back to admin
-        require(
-            seedToken.transfer(admin, seedToken.balanceOf(address(this))),
-            "Seed: should transfer seed tokens to admin"
-        );
-
-        closed = true;
-        paused = false;
+        if(minimumReached){
+            // amount of seed tokens to be distributed = seedAmountAtStart - seedRemainder
+            uint256 seedToTransfer = (seedToken.balanceOf(address(this))).sub(seedAmountAtStart.sub(seedRemainder));
```

Comment:
should we also move here away from the `balanceOf` way?

Reply:
So what I thought was, what if the admin sent some seed by mistake, So what we do is, we only keep seed that has been bought, and send back all the remaining seed. Also I found some problem with calculation now. it should be - ```(seedToken.balanceOf(address(this))).sub(seedAmountAtStart.sub(seedRemainder).add(seedClaimed))```
Or should we go with not using balanceOf().